diff --git a/configure.ac b/configure.ac
index 48eace0..0d6f2d5 100644
--- a/configure.ac
+++ b/configure.ac
@@ -43,6 +43,7 @@ PKG_CHECK_MODULES(libzip, libzip >= $LIBZIP_VERSION)
 PKG_CHECK_MODULES(libcurl, libcurl >= $LIBCURL_VERSION)
 PKG_CHECK_MODULES(openssl, openssl >= $OPENSSL_VERSION)
 PKG_CHECK_MODULES(zlib, zlib >= $ZLIB_VERSION)
+AC_SEARCH_LIBS([pthread_create], [pthread])
 
 GLOBAL_CFLAGS="-Wno-multichar "
 AC_LDADD=""
